docs/howto_firewall.md

```diff
@@ -1,33 +1,40 @@
-|443        |TCP        | HTTPS port used to serve applications through the OpenShift router |
-|6443       |TCP        | HTTPS API port for the MicroShift API |
-|5353       |UDP        | mDNS service to respond for OpenShift route mDNS hosts |
-|30000-32767|TCP/UDP    | Port range reserved for NodePort type of services, can be used to expose applications on the LAN |
+It is mandatory to allow MicroShift pods the access to the internal CoreDNS and API servers.
+> Users may choose to change the pod IP range to be different from the default `10.42.0.0/16` setting. This must be reflected in the firewall configuration.
 
-Additionally, pods need to be able to contact the internal core DNS server.
-Assuming that the pod IP range is `10.42.0.0/16`, run the following command to configure the access.
+|IP Range      |Description|
+|:-------------|:----------|
+|10.42.0.0/16  |Internal CoreDNS access      |
+|169.254.169.1 |MicroShift API Server access |
 
-```bash
-sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
-```
+The following ports are optional and they should be considered for MicroShift if a firewall is enabled.
+
+|Port(s)    |Protocol(s)|Description|
+|:----------|:----------|:----------|
+|80         |TCP        |HTTP port used to serve applications through the OpenShift router |
+|443        |TCP        |HTTPS port used to serve applications through the OpenShift router |
+|6443       |TCP        |HTTPS API port for the MicroShift API |
+|5353       |UDP        |mDNS service to respond for OpenShift route mDNS hosts |
+|30000-32767|TCP/UDP    |Port range reserved for NodePort type of services, can be used to expose applications on the LAN |
 
 ## Firewalld
-The following commands can be used for enabling `firewalld` and opening all the above mentioned ports.
+The following commands can be used for enabling `firewalld` and opening all the above mentioned source IP addresses and ports.
+> Use the appropriate pod IP range if it is different from the default `10.42.0.0/16` setting.
 
 ```bash
 sudo dnf install -y firewalld
 sudo systemctl enable firewalld --now
-sudo firewall-cmd --zone=trusted --add-source=10.42.0.0/16 --permanent
-sudo firewall-cmd --zone=public --add-port=80/tcp --permanent
-sudo firewall-cmd --zone=public --add-port=443/tcp --permanent
-sudo firewall-cmd --zone=public --add-port=6443/tcp --permanent
-sudo firewall-cmd --zone=public --add-port=5353/udp --permanent
-sudo firewall-cmd --zone=public --add-port=30000-32767/tcp --permanent
-sudo firewall-cmd --zone=public --add-port=30000-32767/udp --permanent
+# Mandatory settings
+sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16 
+sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1
+# Optional settings
+sudo firewall-cmd --permanent --zone=public --add-port=80/tcp
+sudo firewall-cmd --permanent --zone=public --add-port=443/tcp
+sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp
```

Comment:
```suggestion
sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp
```

I would move this one to the end, as it's the less likely to be added.

Reply:
Fixed